tests/unit/backend/chalice/api_server/test_v1_collection.py

```diff
@@ -278,3 +279,32 @@ def test__get_collection_uuid__403_not_found(self):
         response = self.app.get(test_url.url, headers=dict(host="localhost"))
         self.assertEqual(403, response.status_code)
         self.assertIn("X-AWS-REQUEST-ID", response.headers.keys())
+
+    def test__post_collection_returns_uuid_on_success(self):
+        test_url = furl(path="/dp/v1/collections/")
+        data = json.dumps({
+            "name": "collection name",
+            "description": "This is a test collection",
+            "contact_name": "person human",
+            "contact_email": "person@human.com",
+            "data_submission_policy_version": "0.0.1",
+            "links": [
+                {"link_name": "DOI Link", "link_url": "http://doi.org/10.1016", "link_type": "DOI"},
+                {"link_name": "DOI Link 2", "link_url": "http://doi.org/10.1017", "link_type": "DOI"}]
+        })
+        response = self.app.post(
+            test_url.url,
+            headers={"host": "localhost", 'Content-Type': "application/json", "Cookie": get_auth_token(self.app)},
+            data=data)
+        self.assertEqual(201, response.status_code)
```

Comment:
see if you can retrieve your newly created collection and verify the owner was set.